x-pack/plugins/monitoring/public/components/elasticsearch/shard_allocation/components/assigned.js

```diff
@@ -5,13 +5,58 @@
  * 2.0.
  */
 
-import { get, sortBy } from 'lodash';
 import React from 'react';
-import { Shard } from './shard';
-import { calculateClass } from '../lib/calculate_class';
+import { get, sortBy } from 'lodash';
+import { css } from '@emotion/react';
 import { EuiFlexGroup, EuiFlexItem, EuiIcon, EuiLink } from '@elastic/eui';
+
+import { Shard } from './shard';
 import { getSafeForExternalLink } from '../../../../lib/get_safe_for_external_link';
 
+const assignedChildrenStyle = (theme) => css`
+  padding-top: ${theme.euiTheme.size.l};
```

Comment:
@mgadewoll Could you elaborate on this? I'd appreciate some resources. Thanks, Lene!

Reply:
[logicalCSS](https://github.com/elastic/eui/blob/351bc8f7be72104c4abbd5ac9b5de5b2c7f95d94/packages/eui/src/global_styling/functions/logicals.ts#L42) is a util to help apply logical css selectors, e.g. instead of writing `padding-top` we should eventually migrate to `padding-block-inline` to account for different writing/order directions. (as we do on EUI side)

While we don't push Kibana devs hard on using this yet, we should be advocates by using those utils, imho.